index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio of Utsargo Roy</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/c488aa6c77.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="wrapper">
+        <div class="navbar col-md-12 space-between align-middle">
+            <a class="site-title" href="#">Welcome</a>
+            <nav>
+                <i class="fa-solid fa-bars"></i>
+            </nav>
+        </div>
```

Comment:
[OPTIONAL] 👍🏽 

- [ ] I think it will be better if you wrap your entire `class="navbar col-md-12 space-between align-middle"` inside of a   `<header>` tag than a `<div>` because it contains the site's title or logo (or both) and the main site navigation.

- [ ] You can as well  [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

What do you think about implementing this suggestion?? 💭 Well, I think it is important. 😄

Reply:
Will do now :)